app/css/Mindfulness.css

```diff
@@ -1,4 +1,15 @@
 /* Mindfulness.css */
+body {
+  font-size: 1rem; /* Base body text size; reflects user/system settings */
+}
+
+h1, .tab-title {
```

Comment:
Same here, h1, should be a global css class.

Reply:
Resolved